tests/test_UI_Flags/.cere/dumps/objs.S

```diff
@@ -1,32 +1,32 @@
+.section s7f4ca620e000, "aw"
```

Comment:
I think this file should not be committed to the repository, can you please remove it.